console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/client/JobGrid.java

```diff
@@ -89,6 +95,14 @@ protected void selectionChangedEvent(GwtJob selectedItem) {
         columnConfigs.add(columnConfig);
 
         columnConfig = new ColumnConfig("userName", MSGS.gridJobColumnHeaderCreatedBy(), 200);
+        columnConfig.setRenderer(new GridCellRenderer<GwtJob>() {
```

Comment:
Uhmmm this column can be namend "CreatedBy" since also the MSGS refers to a `gridJobColumnHeaderCreatedBy` messages. 

I think that `username` column name is a mistake :) 

After rename, the `CreatedByNameCellRenderer` could be used. Please also remove `getUsername()` and `setUsername()` from GwtJob  and use `createtBy` instead.